config/config.development.js.sample

```diff
@@ -0,0 +1,5 @@
+var config = require('./config.global')
+
+config.discord.key = '<Insert Key Here>'
```

Comment:
maybe call it discord.channel_key or something , .key doesnt explain what kind of key.